src/main/java/touro/GameOfLife/GridView.java

```diff
@@ -18,8 +18,8 @@ public GridView(Grid grid) {
     @Override
     protected void paintComponent(Graphics g) {
         super.paintComponent(g);
-        drawGrid(g);
         paintLiveCells(g);
+        drawGrid(g);
```

Comment:
I changed the order of painting here so that the lines are drawn after the live cells, this way they appear between the live cells as well. Otherwise, they got painted over.